src/js/actions/transform.js

```diff
@@ -404,44 +408,59 @@ define(function (require, exports) {
      * @param {boolean} position.relative If true, x and y will be relative to the owner artboard of layer
      * @param {string} refPoint Two character string denoting the active reference point so we know 
      *                 which corner of the layer to move
-     *
+     * @param {object} options Batch play options
+     * 
      * @return {Promise}
      */
-    var setPosition = function (document, layerSpec, position, refPoint) {
+    var setPosition = function (document, layerSpec, position, refPoint, options) {
+        options = _.merge({}, options);
         layerSpec = layerSpec.filterNot(function (layer) {
             return layer.isGroupEnd;
         });
 
         var payload = {
-                documentID: document.id,
-                positions: [],
-                history: {
-                    newState: true,
-                    name: nls.localize("strings.ACTIONS.SET_LAYER_POSITION")
-                }
-            },
-            options = {
-                paintOptions: _paintOptions,
-                historyStateInfo: {
-                    name: nls.localize("strings.ACTIONS.SET_LAYER_POSITION"),
-                    target: documentLib.referenceBy.id(document.id)
-                }
-            };
+            documentID: document.id,
+            positions: [],
+            coalesce: !!options.coalesce,
+            history: {
+                newState: true,
+                name: nls.localize("strings.ACTIONS.SET_LAYER_POSITION")
+            }
+        };
+
+        options = _.merge(options, {
```

Comment:
This will come as no surprise to you, I'm sure ... but I don't like this.  I think it is confusing when action options get merged in to "play" options ... especially when it gets *assigned back to the same variable*